setup_all.sh

```diff
@@ -4,21 +4,31 @@ source config_vars.sh
 
 set -e
 
+ls config/ | grep default | while read cfgfilename; do
+    new_cfgfilename=${cfgfilename::-8}
+    cp -n config/$cfgfilename config/$new_cfgfilename
+done
+
 # Start MySQL server
 ./run_mysql.sh
 
 # Build and run the API
-docker build -t zeeguu-api-core -f docker-files/zeeguu-api-core/Dockerfile .
+docker build --build-arg API_VERSION="1.0" -t zeeguu-api-core -f docker-files/zeeguu-api-core/Dockerfile .
 docker run --net=host -d \
     -e MICROSOFT_TRANSLATE_API_KEY="$MICROSOFT_TRANSLATE_API_KEY" \
     -e GOOGLE_TRANSLATE_API_KEY="$GOOGLE_TRANSLATE_API_KEY" \
     -e WORDNIK_API_KEY="$WORDNIK_API_KEY" \
     --name=zeeguu-api-core zeeguu-api-core
 
+cp -n docker-files/zeeguu-web/apache-zeeguu.conf.default docker-files/zeeguu-web/apache-zeeguu.conf
```

Comment:
can't we do the same thing as we did with the other config files? only if `apache-zeeguu.conf` does not exists then copy the default over it?

Reply:
that would allow one to modify the configuration files, then run setup.py and have the entire infra running

Reply:
That's why I'm passing the `-n` parameter. It should copy the file only if it doesn't exist:
```
-n, --no-clobber
              do not overwrite an existing file (overrides a previous -i
              option)
```

Reply:
ah, neat!